pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -568,7 +568,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 					},
 					{
 						Alert: "LowVirtAPICount",
-						Expr:  intstr.FromString("(num_of_allocatable_nodes > 1) and (num_of_running_virt_api_servers < 2)"),
+						Expr:  intstr.FromString("(num_of_allocatable_nodes > 1) and (kubevirt_virt_api_up_total < 2)"),
```

Comment:
We should probably decide if we want plural or singular: virt_controller and virt_api or virt_controllers and virt_apis.

Reply:
singular

Reply:
singular